ranger/ranger.go

```diff
@@ -43,13 +44,22 @@ type Ranger struct {
 	users     middleware.UserStorer
 }
 
-// New constructs a Ranger from the provided options.
+// Config holds the data that is requried to init a new ranger. This also allow us to
```

Comment:
`requried`

Also, as a trails user, the second sentence is unclear to me. It read more as explanation for a trails dev, maybe move to a comment inside `New` below?